src/core/builders/vite/plugins/multipageModeGlobals.ts

```diff
@@ -0,0 +1,20 @@
+import * as vite from 'vite';
+import { getEntrypointGlobals } from '~/core/utils/globals';
+
+/**
+ * Define a set of global variables for multi-page mode builds.
+ */
+export function multipageModeGlobals(): vite.PluginOption {
```

Comment:
TODO: Combine this and the `libModeGlobals` into one `entrypointGroupGlobals` plugin, and add it to the base config.